packages/node-core/src/configure/configure.module.ts

```diff
@@ -52,7 +52,7 @@ export function yargsToIConfig(yargs: Args, nameMapping: Record<string, string>
   return Object.entries(yargs).reduce((acc, [key, value]) => {
     if (['_', '$0'].includes(key)) return acc;
 
-    if (key === 'network-registry') {
+    if (isMatchFlag(key, ['network-registry'])) {
```

Comment:
This fix is prone to the same error occuring again. If another flag is added and there is now implicit behaviour as to what this does the same issue could be introduced again with another flag. 

I think it would be better to not compare using key and instead use the modified key (`nameMapping[key] ?? camelCase(key)`)